src/utilities/getSmallestNumber/getSmallestNumber.test.js

```diff
@@ -0,0 +1,126 @@
+import { getSmallestNumber } from './getSmallestNumber';
+
+describe('getSmallestNumber()', () => {
+  describe('when provided with valid arrays of numbers', () => {
```

Comment:
You are providing a function with a single array at a time, so I would rephrase that